spec/models/service_template_ansible_playbook_spec.rb

```diff
@@ -225,19 +225,18 @@
       expect(service_template.resource_actions.first.dialog.id).to eq new_dialog_record.id
     end
 
-    it 'uses the existing dialog if :service_dialog_id is passed in' do
+    it 'uses the existing dialog if :dialog_id is passed in' do
```

Comment:
Ahh - I see it now.  `catalog_item_options_three` has it there.  Then cool - Everything looks good.